.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-max-line-length = 99
+max-line-length = 107
```

Comment:
Let's fix those weird lines and keep this at 99, so we don't go to a "broken master" then to fix it.

(And I need to see that this is doable with black ;) )

Reply:
Addressed https://github.com/canonical/charmcraft/pull/315/commits/c43e8a7a0834a15d992455a7f36c7d09ab7b26ec